### DISCUSSION TITLE
Fourm is being down over and over again

### DISCUSSION THREAD

**Discussion by Khnaz35** [opencart/opencart#11514] [General]
Hi Guys, 
I think its time to add on more security layers to the forum.opencart.com

I would suggest following changes into .htaccess:

# Add Security Headers to protect Opencart user
<IfModule mod_headers.c>
	Header set X-XSS-Protection "1; mode=block"
	Header set X-Frame-Options "SAMEORIGIN"
	Header set X-Content-Type-Options "nosniff"
	Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
	# Header set Content-Security-Policy ...
	Header set Referrer-Policy "same-origin"
	Header set Feature-Policy "geolocation 'self'; vibrate 'none'"
</IfModule>

<img width="479" alt="image" src="https://user-images.githubusercontent.com/43812103/183691583-4cadb426-0796-42da-9504-9e75a297b2e0.png">

**Comment by danielkerr** [comment]
will add when i have time.
